collections/vegetation-phenology-and-productivity-parameters-season-1/README.MD

```diff
@@ -82,7 +82,7 @@ The available bands include: 13 parameters and QFLAG
       <td>The time series was not processed.</td>
     </tr>
    </tbody>
-</table> 
+</table>  
 
 ## More information
```

Comment:
If you are anyway already on this maybe remove the linebreaks in lines 91/98 and 105 so the caption is a bit closer to the image (same in the other file).

Reply:
Thanks